rasa/core/schemas/domain.yml

```diff
@@ -27,6 +27,33 @@ mapping:
   responses:
     type: "map"
     allowempty: True
+    matching-rule: 'any'
+    mapping:
+      regex;(.+):
+        type: "seq"
+        required: True
+        sequence:
+        - type: "map"
+          allowempty: True
+          matching-rule: 'any'
+          mapping:
+            regex;(^text$):
```

Comment:
is there a reason to use a regex here? I though that the keys `text`, `image`, `buttons`, `custom` are fixed 🤔
If we replace the regex with fixed values, then `matching-rule: 'any'` is not necessary as well.

Reply:
no I think I got confused 🤷 removing